src/l500/ac-trigger.cpp

```diff
@@ -202,8 +210,17 @@ namespace ivcam2 {
         {
             // User wants to trigger it RIGHT NOW
             // We don't change the actual control value!
-            auto & depth_sensor = _autocal->_dev.get_depth_sensor();
-            if( depth_sensor.is_streaming() )
+            bool is_depth_streaming(false);
+            try
+            {
+                is_depth_streaming = _autocal->_dev.get_depth_sensor().is_streaming();
+            }
+            catch (std::exception const & e)
+            {
+                AC_LOG(ERROR, "EXCEPTION caught in access to device: " << e.what());
```

Comment:
This will essentially trigger the throw in line 230... right? So we didn't save anything by catching. Did we improve anything?

Reply:
Correct, removed